packages/elements/src/tab-bar/index.ts

```diff
@@ -94,15 +94,15 @@ export class TabBar extends BasicElement {
   @property({ type: String, attribute: false })
   public set value (value: string) {
     value = typeof value === 'string' ? value : String(value);
-    const oldValue = this._value;
+    const oldValue = this.oldValue;
     if (value !== oldValue && this.isValidValue(value)) {
-      this._value = value;
+      this.oldValue = value;
```

Comment:
You might not need `oldValue` property if you leave setter and getter to ControlElement where `this. activateTab` can be called during `willUpdate()` lifecycle.